Readme.md

```diff
@@ -140,6 +140,7 @@ For example, to run Pairwise Distance Histogram that outputs a csv file at `foo.
 $ python empress_cli.py histogram examples/heliconius_host.nwk examples/heliconius_parasite.nwk \
                                   examples/heliconius_mapping.mapping --csv foo.csv --histogram bar.pdf --ynorm
 ```
+CSV output has rows with two values each. The first value on each row is the distance (measured using the symmetric set difference between the event sets), and the second value is the number of MPR pairs that are that far apart from each other.
```

Comment:
Do you think it would be a good idea if we also provide an example with an explanation?

Reply:
Actually I don't think this is needed.